tests/test_TFNetworkLayer.py

```diff
@@ -150,6 +150,137 @@ def test_concat_sources_missing_dim():
     session.run(out.output.placeholder)
 
 
+def test_self_att_time_dim_tags():
+  with make_scope() as session:
+    n_in = 128
+    channels = 16
+    n_time = 128
```

Comment:
Use a different number, not the same as `n_in`, such that debugging becomes easier.
(Also, better use prime numbers, such that certain bugs are not hidden.)

Reply:
Done